src/gui/parametersdialog.cpp

```diff
@@ -576,9 +576,18 @@ QString ParametersDialog::getSyncPalErrorText(QString fctCode, ExitCode exitCode
             break;
         case ExitCodeOk:
         case ExitCodeNeedRestart:
-        case ExitCodeInconsistencyError:
+        case ExitCodeLogicError:
+            if (exitCause == ExitCauseFullListParsingError) {
+                return tr("The parsing of the CSV full listing of the remote drive items failed (error %1).<br>"
```

Comment:
Perhaps too technical for an end user.
Maybe only "Special characters such as double quotes, backslashes or line returns can cause kDrive failures."?

Reply:
Agree with the above comment.
+ Move "case ExitCodeInconsistencyError" before "case ExitCodeOk".

Reply:
> Move "case ExitCodeInconsistencyError" before "case ExitCodeOk".
But this is going to shift all the exit codes. Isn't it?

Reply:
Done.